EtlPipelineBlobTrigger/course_stats.py

```diff
@@ -1,105 +1,163 @@
 """Data transformation code for statistical data."""
 
 import json
-import logging
+import inspect
 import os
+import sys
 import unicodedata
 from collections import OrderedDict
 
+CURRENTDIR = os.path.dirname(
+    os.path.abspath(inspect.getfile(inspect.currentframe())))
+sys.path.insert(0, CURRENTDIR)
 
-class Continuation:
-    """Extracts and transforms the continuation course element"""
+from validators import validate_unavailable_reason_code, validate_agg
+
+
+class CourseStats:
+    def get_stats(self, raw_course_data):
+        stats = {}
+
+        continuation = Continuation()
+        employment = Employment()
+        stats['continuation'] = continuation.get_continuation(raw_course_data)
+        stats['employment'] = employment.get_employment(raw_course_data)
+        return stats
+
+
```

Comment:
So I generally find it easier to follow code if the classes and the encapsulated functions are in logical flow from high level to lower levels of abstracted code.

```
class CourseStats:
  ...
class Continuation:
  ...
class Employment:
  ...
class SharedUtils:
  ...
```

Reply:
:+1 Moved SharedUtils to bottom of module.